### PR TITLE
CT: 107 - Update GIDS WEAMS Service to accept two additional data fields #18513

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18513

As a developer, I need to update the GIDS service to accept a WEAMS file that contains two new columns so that the Compare Tool is able to support displaying information for section 107 of the Colmery act.

## Assumptions:
1. This will be updating the "WEAM" CSV Type that is located within the GIDS Service.
     a. The WEAMS file adds the following columns to the current file format:
               - Campus Type
               - Parent Facility Code Id
2. Campus Types include: Main, Branch, Extension
3. "Campus Type" of "Main" does not have a "Parent Facility Code Id"
     a. "Branch" & "Extension" have a "Parent Facility Code Id"
4. Extensions can have parents that are either Branches or Mains. 

## Acceptance Criteria:
1. The Compare Tool will continue to use the file, service, etc and display the information correctly:
     - With the two new columns included
     - Without the two new columns included
2. The data in the new WEAMS columns is persisted with the rest of the WEAMS file data.

### CODE REVIEW

**Review by hughes-dustin — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/backend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by sonntag-adam — APPROVED** [review]

File: app/models/weam.rb [hunk 1]
```diff
@@ -63,7 +63,9 @@ class Weam < ActiveRecord::Base
     'distance learning' => { column: :distance_learning, converter: BooleanConverter },
     'priority enrollment' => { column: :priority_enrollment, converter: BooleanConverter },
     'preferred provider' => { column: :preferred_provider, converter: BooleanConverter },
-    'stem indicator' => { column: :stem_indicator, converter: BooleanConverter }
+    'stem indicator' => { column: :stem_indicator, converter: BooleanConverter },
+    'campus type' => { column: :campus_type, converter: BaseConverter },
```
Comment (jholton) [inline app/models/weam.rb]: Should we add a validation for Campus Type?
Should we raise an error if it's ever blank, or a value other than Main, Branch or Extension?

**Comment by zurbergram** [comment]
Closing because work is being done https://github.com/department-of-veterans-affairs/gibct-data-service/pull/382

**Pull request closed without merging**